pyproject.toml

```diff
@@ -61,6 +61,7 @@ testpaths = ["tests"]
 
 [tool.coverage.run]
 branch = true
+omit = ["src/remarx/app.py", "docs/hooks.py"]
```

Comment:
What is the `docs/hooks.py` you're excluding here? I don't see it in the file list, is this from another branch?